.scss-lint.yml

```
@@ -1,8 +1,13 @@
+scss_files:
+  - 'app/assets/stylesheets/**'
```

We have some stylesheets in the Webpacker packages as well. One or the other of these patterns should be included:

```suggestion
  - 'app/assets/stylesheets/**'
  - 'app/javascript/**'
```

```suggestion
  - 'app/assets/stylesheets/**'
  - 'app/javascript/packages/**'
```

Currently they're only in `packages`, but there's nothing which stops us from using them in any `packs`.

---

Oh thank you, I chose the first for better future-proofing